templates/template/README.md

```diff
@@ -79,24 +78,103 @@ Before submitting any PRs, a Github Workflow will check to see if your package d
 
 {{- end }}
 
+{{- if (eq .Template "source") }}
+### Making Changes
+
+As a developer making changes to a particular package, you will usually follow the following steps:
+
+#### If this is the first time you are adding a package:
+
+```shell
+PACKAGE=<packageName>
+mkdir -p packages/${PACKAGE}
+touch packages/${PACKAGE}/package.yaml
+```
+
+See `packages/README.md` to configure the `packages/${PACKAGE}/package.yaml` file based on the Package that you are planning to add. To make changes, see the steps listed below.
```

Comment:
nit: split sentences into multiple lines

Reply:
addressed in latest, thanks for catching!